app/actions/CommonActions.scala

```diff
@@ -3,17 +3,21 @@ package actions
  import controllers.{Cached, NoCache}
  import play.api.libs.concurrent.Execution.Implicits.defaultContext
  import play.api.mvc._
+ import play.filters.csrf.CSRFCheck
 
- import scala.concurrent.Future
-trait CommonActions {
+ import scala.concurrent.{ExecutionContext, Future}
+final class CommonActions(executionContext: ExecutionContext, cSRFCheck: CSRFCheck, parser: BodyParser[AnyContent]) {
 
-  val StoreAcquisitionDataAction = new ActionBuilder[Request] {
+  val StoreAcquisitionDataAction = new ActionBuilder[Request, AnyContent] {
```

Comment:
in play 2.6 the ActionBuilder also contains a parser for the content.  Usually we just want the default parser which looks at the content type if any, and deserialises with that if applicable.   This is passed in with an overridden field below.